sources/hyper-proxy-shim/README.md

```diff
@@ -0,0 +1,71 @@
+# hyper-proxy
+
+[![MIT licensed](https://img.shields.io/badge/license-MIT-blue.svg)](./LICENSE)
+[![crates.io](http://meritbadge.herokuapp.com/hyper-proxy)](https://crates.io/crates/hyper-proxy)
+
+A proxy connector for [hyper][1] based applications.
```

Comment:
If we're forking this code, I would expect a shorter README.md pointing to the original source, by expanding the credits section. We can probably drop the example.